docs/connect-to-lms.md

```diff
@@ -1,40 +1,30 @@
-## Integrating with a Learning Management System
+## Integrating with a learning management system
 
-GitHub Classroom is continuously adding support for integrating with your institution's Learning Management System (LMS)!
-Connecting GitHub Classroom to a Learning Management System will unlock new capabilities that will streamline and enhance
-the same workflow your used to on GitHub Classroom, such as:
+GitHub Classroom integrates with your institution's learning management system (LMS), such as Brightspace, Canvas, Moodle, and others. Classroom supports:
 
-- Automatic roster provisioning
+- Roster provisioning
 - Assignment creation (Planned)
 - Submission syncing (Planned)
 
-### Supported Learning Management Systems
+Classroom works with any LMS that uses the Learning Tools Interoperability (LTI) standard. LTI is an industry-standard protocol certified by the Instructional Management System Global Learning Consortium. Using LTI keeps your information safe and secure.
 
-GitHub Classroom integrates with your Learning Management via the industry-standard LTI protocol and is certified by the
-Instructional Management System Global Learning Consortium, so you can be confident your information is safe and secure. Any
-Learning Management System which is compliant with the LTI standard is compatible with GitHub Classroom, such as
-Brightspace, Canvas, Moodle, and many more!
-
-Note: Blackboard does not currently support automatically importing a course roster. We are working to support Blackboard in the near future!
+**Note**: Blackboard does not currently support automatically importing a course roster. We are working to support Blackboard in the near future!
```

Comment:
The lack of support currently lies with Classroom not supporting Blackboard, though the question we're trying to solve in the meantime is: are we not supporting Blackboard right now because Blackboard doesn't adhere to LTI standard in the way we expect and need to workaround it on our end, or because Blackboard simply does not support the LTI standard in a way that an instructor can feasibly integrate with GitHub Classroom without wasting the greater part of day attempting to get an IT administrator to install it for them.

For now, it's the latter.

Reply:
That's very helpful! It sounds like there's some use in retaining the ambiguity. What do you think of this revision?

```suggestion
**Note**: Blackboard and GitHub Classroom course rosters don't work together yet. We are working to support Blackboard in the near future!
```

Reply:
:+1: